driver-core/src/main/com/mongodb/ConnectionString.java

```diff
@@ -220,6 +221,11 @@
  * <li>{@code zlibCompressionLevel=integer}: Integer value from -1 to 9 representing the zlib compression level. Lower values will make
  * compression faster, while higher values will make compression better.</li>
  * </ul>
+ * <p>SRV configuration:</p>
+ * <ul>
+ * <li>{@code srvServiceName=string}: The SRV service name.</li>
+ * <li>{@code srvMaxHosts=number}: The maximum number of hosts from the SRV record to connect to.</li>
```

Comment:
Should `MongoClientURI`/`MongoClientOptions` also support these options, or are we not adding support for any new options to those classes?